flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/dataset/DataSetUserDefinedFunctionITCase.scala

```diff
@@ -117,6 +117,187 @@ class DataSetUserDefinedFunctionITCase(
   }
 
   @Test
+  def testDynamicSchema(): Unit = {
```

Comment:
Yes, I agree with you. I've updated the consistently patch. I will refine these tests very soon today.